azerothcore/azerothcore-wotlk
# fix(DB/Smart Scripts) - Door Lever - On Game Object State Changed.

**Changes Proposed:**

-  Fix all door's lever, Now Opening and Close all Door's - working.

**Issues Addressed:**

- Closes : Solving the problem with all doors regarding opening and closing.

**Tests Performed:**

- Inside the game
- Windows Server 2022 Datacenter

**How to Test the Changes:**
- .tele deadmines, kill first boss and test lever (door open and close)

**How to Test AzerothCore PRs:**

When a PR is ready to be tested, it will be marked as [WAITING TO BE TESTED].

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

REMEMBER: when testing a PR that changes something generic (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but especially check that the PR does not cause any regression (i.e. introducing new bugs).

For example: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but we should test Y and Z as well.
